### PR TITLE
Edit page mockup done

### PR DESCRIPTION
**Pull request by randomzebra**
Edit page done, but isn't functional yet. There needs to be way to interface with EventModels (search for event by ID, pull with getters and setters), which is simple to do but out of scope for this PR

### CODE REVIEW

File: client/client-app/EditEvent.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
+import { useState } from "react";
+import { SafeAreaView, Text, View, TextInput, Image, TouchableOpacity, ScrollView} from "react-native";
+import styles from "./styles";
+import { EVENTS } from "./Events"
+
+
+const EditEvent = ({navigation, id}) => {
+
```
Comment (johngarrett) [inline client/client-app/EditEvent.js:8]: nit: blank lines

File: client/client-app/EditEvent.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+                    navigation.navigate('TabRoot', {name: 'Jane'})
+                }
+            >
+
+                <Text style={styles.eventTextSecondary}>
+                    {"Test + " + id}
+                </Text>
+            </TouchableOpacity>
+            {/* <View>
+                <Image source={require('../client-app/assets/images/office_potluck.jpg')} style={{height: imgHeight, width: 365}}>
```
Comment (johngarrett) [inline client/client-app/EditEvent.js:38]: nit: comments in main

File: client/client-app/models/EventModel.js [hunk 3]
```diff
@@ -4,8 +4,33 @@ class EventModel {
         this.name = name;
         this.date = date;
         this.time = time;
-        this.imgSrc = imgSrc;
+        this.imgSrc = imgSrc; // When we begin to add images, we should probably move 
+                              // image representation away from a file path. Maybe a database ID?
         this.description = description;
     }
+
+    get id() { return this._id; }
```
Comment (johngarrett) [inline client/client-app/models/EventModel.js:12]: Is there a reason we're using getters and setters? It feels a bit Java-y

Comment (randomzebra) [reply client/client-app/models/EventModel.js:12]: Nah your right, we can probably just access the individual components. I can make a commit to main later.

**Comment by manavk-p** [comment]
Great work on this PR, could you let me know when the PR interfacing with event model is out? I'd like to take a look at that as well.

**Pull request merged**